hubmap/query_app/migrations/0008_delete_atacquant_delete_codexquant_and_more.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 4.0.6 on 2022-08-03 12:55
```

Comment:
Did we start this app with Django 3.2? Out of curiosity, were there any backwards-incompatible changes in 4.0 that affected us?

Reply:
I think so, and none that affected us, as I recall.